src/common/_modules/function_admin/main.tf

```diff
@@ -0,0 +1,65 @@
+resource "azurerm_resource_group" "function_admin_itn_rg" {
+  name     = "${var.project_itn}-funcadm-rg-01"
+  location = var.location_itn
+
+  tags = var.tags
+}
+
+module "function_admin_dx" {
+  source  = "pagopa-dx/azure-function-app/azurerm"
+  version = "~> 2.0"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location_itn
+    app_name        = "funcadm"
```

Comment:
just `admin`, func it's already present in the naming convention
Otherwise that function name will be:
`io-p-itn-funcadmin-func-01`